src/Stratum/handlers/base.ts

```diff
@@ -0,0 +1,21 @@
+
+export interface StratumRequest {
+  id: number;
+  method: string;
+  params: any[];
+}
+
+export interface StratumResponse {
+  id?: number;
+  result: any;
+  error?: any[];
+}
```

Comment:
There are types for request and responses already

https://github.com/ProofOfWorkCompany/boost-stratum/blob/develop/src/Stratum/request.ts
https://github.com/ProofOfWorkCompany/boost-stratum/blob/develop/src/Stratum/response.ts

Reply:
I think it would be better to `export interface` rather than `export type`